lavamoat/browserify/beta/policy.json

```diff
@@ -187,9 +187,23 @@
         "crypto": true
       },
       "packages": {
+        "@ethereumjs/tx>ethereum-cryptography>@noble/curves": true,
         "@ethereumjs/tx>ethereum-cryptography>@noble/hashes": true,
-        "@ethereumjs/tx>ethereum-cryptography>@scure/bip32": true,
-        "@metamask/message-signing-snap>@noble/curves": true
```

Comment:
See [snap changelog](https://github.com/MetaMask/message-signing-snap/blob/main/CHANGELOG.md#changed) for dependency changes, hence lavamoat policy update.